pkg/services/ngalert/api/api_alertmanager_base.go

```diff
@@ -18,14 +18,15 @@ import (
 )
 
 type AlertmanagerApiService interface {
-	RouteCreateSilence(*models.ReqContext, apimodels.SilenceBody) response.Response
```

Comment:
The big diff here is mostly due to a re-order, I've made it a bit easier to understand by it.